### ISSUE TITLE
Feedback from initial review by practicing Data Scientist (JL)

### ISSUE THREAD

**Issue by cbituin** [cbituin/nfl-big-data-bowl-2022#1]
Nice notebook. Well organized and clear.

Observations:

1. In cell 28, I believe you refit your ColumnTransformer on your test data. All preprocessing steps are set on your training data, then just call transform  (not fit_transform).

2. It's useful to keep your features in a list, so that you can just say fg_test[features]. Setting this list up in your training step prevents you from ever having your train and test features out of sync.

3. Accuracy metrics. Think about the distribution of your target class. What's the accuracy of something that predicts 1 all the time? Have some idea of baseline/bias and choose an appropriate metric.

4. In cell 6, why do you copy those dataframes? Shouldn't be an issue here, but once you star throwing around big pandas objects, you definitely need to keep an eye on memory utilization.

5. Line graphs show some kind of progression (usually through time) or relationship between things along the x axis. Is there any relationship between kicker 300 and 301? CIN and CLE? These should probably be bar charts.

6. You should star to think about using the object oriented matplotlib style. It's going to give you a lot more control. https://matplotlib.org/matplotblog/posts/pyplot-vs-object-oriented-interface/

**Comment by cbituin** [comment]
1. changed to `fg_predict = full_pipeline.transform(fg_prepared_preds)`
2. abstracted `features` list and replaced accordingly
4. initially copied with the intent to manipulate. removed since copies were no longer used. 
5. updated charts

**Comment by cbituin** [comment]
items 3 and 6 will need more research
